BrainPortal/cbrain_plugins/cbrain-plugins-base/userfiles/file_collection/views/_plain_file_list_row.html.erb

```diff
@@ -41,7 +41,7 @@
     <% else %>
       <%= image_tag "/images/file_icon.png" %>
     <% end %>
-    <% if file.size > 0%>
+    <% if file.size > 0 && file.symbolic_type != :directory %>
```

Comment:
Instead of comparing as `not a directory`, compare to `is a plain file`, to be safer